timescaledb/getting-started/create-cagg/create-cagg-basics.md

```diff
@@ -0,0 +1,119 @@
+# Creating continuous aggregates
+
+Now that you've been introduced to continuous aggregates, create your own continuous
+aggregate from your data. 
+
+## Step 1: Create an aggregate query to use in your continuous aggregate
+
+The data we are using for this tutorial is second-by-second (tick) data for stock trades. 
+A popular aggregate pattern used for analyzing stock data is called a [candlestick][candlestick]. 
+Generally, candlestick charts use 4 different aggregations over a specific interval 
+of time (ie. 1-minute, 5-minute, or 1-day aggregates):
+
+* `high`: highest stock price per interval
+* `open`: opening stock price per interval
+* `close`: closing stock price per interval
+* `low`: lowest stock price per interval 
+
+For this exmaple query, the [`time_bucket()`][time-bucket] function interval will be 1 day. The 
+`high` and `low` values can be found by using the PostgreSQL [`MAX()`][max] and [`MIN()`][min] 
+functions. Finally, the `open` and `close` values can be found by using the [`first()`][first] 
+and [`last()`][last] functions.  
+
+```sql
+SELECT
+	time_bucket('1 day', "time") AS day,
+	symbol,
+	max(price) AS high,
+	first(price, time) AS open,
+	last(price, time) AS close,
+	min(price) AS low
+FROM stocks_real_time srt
+GROUP BY day, symbol;
+```
```

Comment:
```suggestion
GROUP BY day, symbol
ORDER BY day DESC, symbol;
```
```
Totally missed this, but we should provide best practices. The other query will be very random on the time column ordering when a user runs it (probably).

Also, I think it would be helpful to see an example couple of rows.